mapreduce/tests/test_mapreduce_integration_e2e.py

```diff
@@ -10,13 +10,24 @@
 
 @pytest.mark.integration
 @pytest.mark.usefixtures("dd_environment")
-def test_integration(aggregator, check, instance):
+def test_integration(aggregator, check, instance, datadog_agent):
     check = check(instance)
+    check.check_id = 'test:123'
     check.check(instance)
+
     for metric in common.ELAPSED_TIME_METRICS:
         aggregator.assert_metric(metric)
     assert_metrics_covered(aggregator)
 
+    version_metadata = {
+        'version.raw': '2.7.1',
+        'version.scheme': 'semver',
+        'version.major': '2',
+        'version.minor': '7',
+        'version.patch': '1',
+    }
+    datadog_agent.assert_metadata('test:123', version_metadata)
```

Comment:
I think it's better to have a separate test. When there is a failure, it's easier to diagnose what's not working.